packages/dds/README.md

```diff
@@ -0,0 +1,15 @@
+# Distributed Data Structures
+
+Fluid's Distributed Data Structures (DDSes) packages live here.
+Documentation for using them can be found [here](../../docs/content/docs/build/dds.md).
+
+This documentation (below) is for people working on DDS implementations.
+This document is incomplete and extensions to it are welcome.
+
+## <a name="trailing-ops"></a>Trailing Ops and Compatibility
+
+When initializing a DDS instance from an existing document, first the summary is loaded, and then any ops since that summary are applied.
+These ops are called "Trailing Ops", and they can get persisted as part of the server side summarization when a session ends between summaries.
+Since Fluid supports being used in a mode (and it typically used in this mode) where there is no service that summarizes documents to bake these trailing ops into the DDS level summaries,
```

Comment:
Are you looking to draw a distinction here about the mechanism used to store the trailing ops?  If so, maybe call out the possibilities more explicitly, e.g.  "Fluid supports two ways a service might store trailing ops:  either as explicit deltas using DeltaStorage, or baked into the summary itself".

Also -- this document states it is for DDS implementers -- what do they need to know about this nuance?  Seems like this all happens behind the scenes and a DDS implementer doesn't care?

Reply:
Oh, the trailing ops could be stored *still as independent ops* within a summary? I thought that "baked into the summary" meant "they are already considered when generating the snapshot that is the latest summary", which to me meant that at that point the ops as separate entities did not exist anymore (or at least weren't relevant anymore). If the former is true, then I guess I'm an example that making that a bit more explicit could be good 😄 .

Reply:
I have rephased this to hopefully be clearer.

I have no idea what DeltaStorage is, so I'm not addressing it here. It seems unnecessary for making the point that we need back compat. If you understand it, and think it would be useful to cover in this document, feel free to add it :).